src/Application/HttpModels/Donation.php

```diff
@@ -34,6 +34,4 @@ class Donation
     public string $projectId;
     public ?float $tipAmount = null;
     public ?bool $tipGiftAid = null;
-    public ?string $cardBrand = null;
-    public ?string $cardCountry = null;
```

Comment:
@NoelLH while I'm here do you think I should remove `$donationMatched` and `$projectId`? PhpStorm says they're unused, I'm not sure what they were fore.

Reply:
These properties are already removed from the client side model in https://github.com/thebiggive/donate-frontend/commit/5dbf0aa5ced which is in prod.

Reply:
I'm struggling to quickly figure this one out. To be clear, `donationMatched` and `projectId` are _not_ removed from the general client side Donate model and are used there, but it does seem correct that in this specific PHP model they're not sent. (They are in `DonationCreate` though.) I'm not sure if they were ever needed or if we got 2 similar models conflated.

Reply:
OK, lets leave them here for now.